SimeonInstaller.ps1

```diff
@@ -969,6 +969,73 @@ CRLFOption=CRLFAlways
         }
 
         Install-SimeonTenantPipeline -Organization $Organization -Project $Project -Name $Name -Credential $Credential -PipelineVariables $PipelineVariables @environmentArgs
+
+        Install-SimeonTenantLibrary -Organization $Organization -Project $Project
+    }
+
+    <#
+    .SYNOPSIS
+    Creates/updates a shared library for a tenant in Azure DevOps
+    #>
+    function Install-SimeonTenantLibrary {
+        [Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSReviewUnusedParameter', '', Scope = 'Function')]
+        [CmdletBinding()]
+        param(
+            # The Azure DevOps organization name
+            [ValidateNotNullOrEmpty()]
+            [string]$Organization,
+            # The project name in DevOps
+            [ValidateNotNullOrEmpty()]
+            [string]$Project = 'Tenants'
+        )
+
+        $token = Get-SimeonAzureDevOpsAccessToken -Organization $Organization -Project $Project
+        $apiBaseUrl = "https://dev.azure.com/$Organization/$Project/_apis"
+        $restProps = @{
+            Headers = @{
+                Authorization = "Bearer $token"
+                Accept = "application/json;api-version=6.0-preview.2"
+            }
+            ContentType = 'application/json'
+        }
+
+        $variableGroupsApi = "$apiBaseUrl/distributedtask/variablegroups"
+        $variableGroups = irm @restProps $variableGroupsApi -Method Get
+
+        $createVariableGroup = $true;
+        if ($variableGroups) {
+            foreach ($variableGroup in $variableGroups) {
+                if ($variableGroup.name -eq 'SyncVariables') {
+                    $createVariableGroup = $false;
+                    break;
+                }
+            }
+        }
+
+        if ($createVariableGroup) {
+            $newVariableGroup = irm @restProps $variableGroupsApi -Method Post -Body (@{
+                    description = 'Simeon Sync shared variables'
+                    name = 'SyncVariables'
```

Comment:
on second thought would actually just call the variable group "Sync" - "SyncVariables" seems redundant since it's a variable group already